pkg/services/grafana-apiserver/auth/authorizer/org/org_id.go

```diff
@@ -20,7 +20,10 @@ type OrgIDAuthorizer struct {
 }
 
 func ProvideOrgIDAuthorizer(orgService org.Service) *OrgIDAuthorizer {
-	return &OrgIDAuthorizer{log: log.New("grafana-apiserver.authorizer.orgid"), org: orgService}
+	return &OrgIDAuthorizer{
+		log: log.New("grafana-apiserver.authorizer.orgid"),
```

Comment:
should we add the log even if we don't use it?  just to make sure the pattern is clear?